inc/namespace.php

```diff
@@ -854,6 +860,22 @@ function get_cloudfront_client() : CloudFrontClient {
 	] );
 }
 
+/**
+ * Get a configured CloudWatch logs client.
+ *
+ * @return CloudWatchLogsClient
+ */
+function get_cloudwatch_logs_client() : CloudWatchLogsClient {
+	static $cloudwatch_logs_client;
+	if ( $cloudwatch_logs_client ) {
+		return $cloudwatch_logs_client;
+	}
+	$cloudwatch_logs_client = Altis\get_aws_sdk()->createCloudWatchLogs([
+		'version' => '2014-03-28',
+	]);
```

Comment:
Needs further spacing (not sure why that's not sniffed)

```suggestion
	$cloudwatch_logs_client = Altis\get_aws_sdk()->createCloudWatchLogs( [
		'version' => '2014-03-28',
	] );
```

Should we consider bumping this API version?